sqtom/fitting_1d.py

```diff
@@ -110,10 +110,12 @@ def marginal_calcs_1d(pd_data, as_dict=True):
     n = np.arange(intn)
     nmean = pd_data @ n
     nmean2 = pd_data @ n ** 2
+    nmean3 = pd_data @ n ** 3
```

Comment:
What is the order of operations here? Do we want `(pd_data @ n) ** 3` or `pd_data @ (n ** 3)`? I note you've explicitly written the latter in `marginal_calcs_2d`, but it's not immediately obvious to me why we wouldn't want to cube the entire thing.

Reply:
Nm, I misunderstood what these expressions are. The probabilities tell how many ones, twos, threes, etc. there are, and so need to be weighted by one, two, three, etc to get `nmean`.

Reply:
Thus, we want `pd_data @ (n ** 3)`, and I believe that is the order that we get without brackets. Still, may as well update either `marginal_calcs_1d` to add brackets or `marginal_calcs_2d` to remove brackets, just for consistency.